cmd/istio-cni-daemonset/main.go

```diff
@@ -0,0 +1,363 @@
+// Copyright 2019 Istio Authors
```

Comment:
nit?:
This is a command line, could be used from istioctl too.
Can you move all the functional code to `pkg` and just *use* that code from here?
There is nothing in the main code that says daemon ...

Reply:
Yep, I can do so. I was just keeping it in a single file for simplicity, but that'd probably be a better way to architect it.

Reply:
We already have a CNI daemonset, are users expected to run a second daemonset as well? maybe I missed context here but it seems like we should just have one daemonset that does both functionality

Reply:
This should be run as (part of the ) cni daemonset or it can be run as a command line utility.